src/sql/migrations/20220704130411_add_delete_sample_file_trigger.js

```diff
@@ -0,0 +1,64 @@
+const getTemplateValues = (dbEnv) => {
+  let body = '';
+  let header = '';
+
+  if (['production', 'staging'].includes(dbEnv)) {
+    header = `
+      CREATE EXTENSION IF NOT EXISTS aws_commons;
+      CREATE EXTENSION IF NOT EXISTS aws_lambda;
+    `;
+
+    const triggerLambdaARN = `arn:aws:lambda:${process.env.AWS_REGION}:${process.env.AWS_ACCOUNT_ID}:function:delete-sample-file-lambda-${dbEnv}`;
+    body = `PERFORM aws_lambda.invoke('${triggerLambdaARN}', row_to_json(OLD));`;
+  }
+
+  return { body, header };
+};
+
+const createDeleteSampleFileTriggerFunc = (env) => {
+  const { header, body } = getTemplateValues(env);
+
+
+  const template = `
+      ${header}
+
+      CREATE OR REPLACE FUNCTION public.delete_sample_file_on_sample_delete()
```

Comment:
Is this name ok? It is supposed to delete `delete_from_s3_on_sample_file_delete`, right? (Or `clean_up_s3_on_sample_file_delete` or some other name)

When I first read this I thought it would delete a `sample_file` row when a `sample` row was deleted, so I would try to keep the name as unambiguous as possible.